lib/awful/tooltip.lua

```diff
@@ -244,27 +550,26 @@ tooltip.new = function(args)
         self.timer:connect_signal("timeout", self.timer_function)
     end
 
+    local fg = beautiful.tooltip_fg or beautiful.fg_focus or "#000000"
```

Comment:
@Elv13 
This breaks for the xresources theme: it seems to use the same fg as bg.
Forcing this to be "#000000" makes the text of tooltips appear.